pypxe/tftp.py

```diff
@@ -59,20 +61,38 @@ def filename(self, message):
         '''
         return message.split(chr(0))[0]
 
-    def notFound(self, address):
+    def tftpError(self, address, code=1, message="File Not Found"):
         '''
-            short int 5 -> Error
-            short int 1 -> File Not Found
-
-            This method sends the message to the client
+            short int 5 -> Error Opcode
+            This method sends the error message to the client
+
+            Error codes:
+            Value     Meaning
+            0         Not defined, see error message (if any).
+            1         File not found.
+            2         Access violation.
+            3         Disk full or allocation exceeded.
+            4         Illegal TFTP operation.
+            5         Unknown transfer ID.
+            6         File already exists.
+            7         No such user.
         '''
-        response =  struct.pack('!H', 5) #error code
-        response += struct.pack('!H', 1) #file not found
-        response += 'File Not Found'
+        response =  struct.pack('!H', 5) # error opcode
+        response += struct.pack('!H', code) # error code
+        response += message
         response += chr(0)
-        self.logger.debug("TFTP Sending 'File Not Found'")
+        self.logger.debug("TFTP Sending '{code}: {message}'".format(code = code, message = message))
```

Comment:
Can this be moved somewhere such that a `File Not Found` error message is accompanied with the requested filename?